packages/@okta/vuepress-site/docs/api/resources/event-hooks/index.md

```diff
@@ -0,0 +1,629 @@
+|-------------|--------------------------------------------------------------------------------------------|------------|-------------------------------------------|----------|
+| Event Hook | A valid Event Hook object, specifying the details of the event hook you are registering. | Body       | [Event Hook object](#event-hook-object) | TRUE     |
+
+#### Response Parameters
+
+The response is an [Event Hook object](#event-hook-object) representing the event hook that was registered. The `id` property returned in the response serves as the unique ID for the registered event hook, which you can specify when invoking other CRUD operations.
+
+##### Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+    "name" : "My Test Event Hook",
+    "events" : {
+        "type" : "EVENT_TYPE",
+        "items" : [
+            "user.lifecycle.create",
+            "user.lifecycle.activate"
+        ]
+    },
+    "channel" : {
+        "type" : "HTTP",
+        "version" : "1.0.0",
+        "config" : {
+            "uri" : "https://www.example.com/eventHooks",
+            "headers" : [
+                {
+                    "key" : "X-Other-Header",
+                    "value" : "some-other-value"
+                }
+            ],
+            "authScheme" : {
+                "type" : "HEADER",
+                "key" : "Authorization",
+                "value" : "${api-key}"
+            }
+        }
+    }
+}' "https://{yourOktaDomain}/api/v1/eventHooks"
+```
+
+> Note: `X-Other-Header` is an example of an additional optional header, with its value specified as `some-other-value`. For each optional header, you choose the name and value you wish Okta to pass to your external service.
+
+##### Response Example
+
+```json
+{
+    "id": "whor0dvWvbMQJHZCM0g3",
+    "name" : "My Test Event Hook",
+    "status": "ACTIVE",
+    "verificationStatus": "UNVERIFIED",
+    "events" : {
+        "type" : "EVENT_TYPE",
+        "items" : [
+            "user.lifecycle.create",
+            "user.lifecycle.activate"
+        ]
+    },
+    "channel" : {
+        "type" : "HTTP",
+        "version" : "1.0.0",
+        "config" : {
+            "uri" : "https://www.example.com/eventHooks",
+            "method" : "POST",
+            "headers" : [
+                {
+                    "key" : "X-Other-Header",
+                    "value" : "some-other-value"
+                }
+            ],
+            "authScheme" : {
+                "type" : "HEADER",
+                "key" : "Authorization",
+            }
+        }
+    },
+    "created": "2018-05-15T01:23:08.000Z",
+    "lastUpdated": "2018-05-15T01:23:08.000Z"
+}
+```
+
+> Note: The `channel.authScheme.value` property is not returned in the response. You set it in your request, but it is not exposed in any responses. 
```

Comment:
I was initially confused by this, because there is no `value` property shown above on `channel.authScheme`.